### DISCUSSION TITLE
Use of a POE-Cam not possible - Can you recommend a Wlan-Cam?

### DISCUSSION THREAD

**Discussion by hpoperator86** [blakeblackshear/frigate#9307] [Ask A Question]
hello dear community,

I have just set up my HomeAssistant and am now faced with the situation that I need/want to buy (at least) one outdoor camera. I have read and internalized the recommendations in the Frigate documentation. Unfortunately, I live in a rented house and as it's a new build, I'm not allowed to drill holes through the house, windows, beams, roof gables and so on. It's extremely well insulated and so the landlord is worried that drilling holes might change anything. I am also not allowed to lay any cables through the roof insulation or roof tiles. In short: I can't pass a POE cable through to the outside and therefore can't use a POE camera.
I would then have to rely on a WLAN cam. Although this is not recommended in the documentation, can you recommend a model that is good and reasonably priced?

Best regards and have a nice evening!

**Comment by nwish** [comment]
So far, my Reolink and Amcrest Wifi cameras have been pretty reliable in conjunction with Frigate.  I’d say just pick whatever camera has the features you want that’s in your budget and run with it, so long as it has an accessible stream, but be prepared to deal with some potential connection stability issues.

**Comment by FrigateEnthusiast** [comment]
Hard to give specific advice without knowing what your wifi specs are.

You want the wifi cam to be operating in "rarified air". And that typically means 5GHZ (802 ac) with the channel set to something like 153, and the AP positioned on the wall opposite the camera. A 80MHZ channel width is a nice touch.

**Comment by hpoperator86** [comment]
Thank you very much for your reply.

I understand that, of course. But I thought that you might already have some models "up your sleeve", precisely because the question has certainly been asked more often.

I currently have a model from Wansview (with solar-module) on the list, which would have the advantage that you wouldn't have to pull a power cable:
https://wansview.com/b4

would this model meet the requirements?

Of course, I would also be open to pure wifi models with a separate power supply. I could use the power supply from the garage, at least for my entrance area and my garden. In front of the house (front garden), where I also intend to install a camera, the power supply to the camera will also be almost impossible (maybe it will have to be a Blink here?).

**Reply by FrigateEnthusiast** [reply]
"_I  understand that, of course_" - Well I try not to assume anything...

That is not a 5GHZ capable camera. So if you live in a dense wifi environment, it will be be very glitchy/spotty. Think - "motion being detected comparing frame to prior frame" and your cam dropping frames like crazy every time your neighbor's kids (or yours) all start watching the latest tictoc videos.
